README.md

```diff
@@ -10,6 +12,17 @@
 [![Slack](https://img.shields.io/badge/%23eventing-white.svg?logo=slack&color=522a5e)](https://knative.slack.com/archives/C9JP909F0)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/5913/badge)](https://bestpractices.coreinfrastructure.org/projects/5913)
 
+## What is Knative Eventing?
+
+Knative Eventing is a collection of APIs that enable you to use an [event-driven architecture](https://en.wikipedia.org/wiki/Event-driven_architecture){target=_blank} with your applications. You can use these APIs to create components that route events from event producers to event consumers, known as sinks, that receive events. Sinks can also be configured to respond to HTTP requests by sending a response event.
```

Comment:
It looks like the `{target=_blank}` isn't working properly with however github renders markdown

Reply:
ok, that text is copy/paste from the website. So, that functionality maybe not working on MD/Github :smile: